openedx/core/djangoapps/appsembler/settings/settings/devstack_common.py

```diff
@@ -27,11 +27,13 @@ def plugin_settings(settings):
             if cache_key != 'celery':  # NOTE: Disabling cache breaks things like Celery subtasks
                 settings.CACHES[cache_key]['BACKEND'] = 'django.core.cache.backends.dummy.DummyCache'
 
-    django_extensions_app_name = 'django_extensions'
-    if django_extensions_app_name not in settings.INSTALLED_APPS:
-        settings.INSTALLED_APPS += [
-            django_extensions_app_name,
-        ]
+    if not getattr(settings, 'TAHOE_NUTMEG_TEMP_SKIP_TEST'):
+        raise ValueError('Nutmeg Code Merge 1/6 - Note 10')
+        # django_extensions_app_name = 'django_extensions'
+        # if django_extensions_app_name not in settings.INSTALLED_APPS:
+        #     settings.INSTALLED_APPS += [
+        #         django_extensions_app_name,
+        #     ]
```

Comment:
ditto, don't comment code but have it enabled and possibly broken in production:

```suggestion
        # TODO: Fix Nutmeg Code Merge 1/6 - Note 10, until then this is 
        #             a temp workaround for appsembler setting tests
        django_extensions_app_name = 'django_extensions'
        if django_extensions_app_name not in settings.INSTALLED_APPS:
            settings.INSTALLED_APPS += [
                django_extensions_app_name,
            ]
```

Reply:
Done